meedan/check-web
# Fix/cv2-3693: cannot restore or mark as not spam within items

## Description

Re-adding: restore from trash/mark as not spam" button on item page and removing deprecated code related to "confirm" items

Reference: CV2-3693

## Type of change


## How has this been tested?

- Send an item to the trash
- Go to the item page
Check that the button to "Restore From Trash" is displayed 
- click on the button
Check that the item is restored 

- Mark an item as Spam
- Go to the item page
Check that the button to "Mark as Not Spam" is displayed 
- click on the button
Check that the item is restored 

## Checklist
